EnglishFilter/sections/specific.txt

```diff
@@ -1021,8 +1021,6 @@ upload-4ever.com##a[onclick^="window.open("] ~ a[style*="text-align: left;"][sty
 ||banner-api.grouphfm.com/banner/
 ||zoophiliahentai.com/uploads/banners/
 olx.ro,olx.kz,olx.pl,olx.pt,olx.ua,olx.uz,olx.in,olx.com##div[style="min-height: 120px; display: block;"]
-http$third-party,~image,domain=imageking.xyz|imgkoi.xyz|imgair.net|imgko.xyz|imglin.xyz|imgtigr.xyz|imgkr.xyz|imglion.xyz|imgsupa.xyz|imgviv.xyz|imgbird.xyz|imgfsh.xyz|imgtiger.xyz|imgvivo.xyz|imgbrd.xyz|imgfish.xyz|imgsup.xyz|imgkor.xyz|imgfrost.net|imghb.xyz|imgkaida.xyz|imgex.xyz|imgopp.xyz|imgbigo.xyz|imgez.xyz
-/h1j.js$domain=imageking.xyz|imgkoi.xyz|imgair.net|imgko.xyz|imglin.xyz|imgtigr.xyz|imgkr.xyz|imglion.xyz|imgsupa.xyz|imgviv.xyz|imgbird.xyz|imgfsh.xyz|imgtiger.xyz|imgvivo.xyz|imgbrd.xyz|imgfish.xyz|imgsup.xyz|imgkor.xyz|imgfrost.net|imghb.xyz|imgkaida.xyz|imgex.xyz|imgopp.xyz|imgbigo.xyz|imgez.xyz
```

Comment:
Why were these rules removed?

Reply:
There is still a detection. Is the script obfuscated? I seem not to understand it.

Reply:
@zloyden I will add these back due to regex not working.

Reply:
Ok. But it's quite difficult to work with a list when you add a domain in random order, especially when we have a lot of them. I've done this one recently, so let's keep it that way. New domains add left to both lists (url, cosmetics) in the future.

```
pikupalate.com,kekolangti.com,imgflyer.xyz,imgez.xyz,imgbigo.xyz,imgopp.xyz,imgex.xyz,imgkoi.xyz,imageking.xyz,imgweng.xyz,cloudgallery.net,imgair.net,imgkaida.xyz,imgfrost.net,imghb.xyz,imgkor.xyz,imgsup.xyz,imgfish.xyz,imgbrd.xyz,imgvivo.xyz,imgtiger.xyz,imgfsh.xyz,imgbird.xyz,imgviv.xyz,imgsupa.xyz,imglion.xyz,imgkr.xyz,imgair.net,imgtigr.xyz,imglin.xyz
```